src/cond.rs

```diff
@@ -11,7 +14,7 @@ pub struct Cond {
 
 impl Cond {
     pub fn new() -> Result<Cond> {
-        let (rfd, wfd) = pipe2(OFlag::O_CLOEXEC)?;
+        let (rfd, wfd) = pipe2(OFlag::O_CLOEXEC)?; //Sets as close-on-execution
```

Comment:
Can you put a space after `//` following other comments?

Reply:
Also putting the comment on a line above would be more inline with current formatting.

Reply:
Updated, also added a one-sentence description for src/command module